xbrowse_server/base/management/commands/add_individuals_to_project.py

```diff
@@ -50,18 +48,46 @@ def handle(self, *args, **options):
             else:
                 vcf = open(vcf_path)
             indiv_id_list = vcf_stuff.get_ids_from_vcf(vcf)
-            self.__add_individuals_to_phenotips(indiv_id_list)
             sample_management.add_indiv_ids_to_project(project, indiv_id_list)
 
+        individual_details=None
         if options.get('ped'):
             fam_file = open(options.get('ped'))
-            sample_management.update_project_from_fam(project, fam_file)
+            individual_details = sample_management.update_project_from_fam(project, fam_file)
 
+        #add to phenotips
+        if options.get('create_phenotips_patients'):
+          #Favor PED file rich information VCF file minimum information to create patients
+          if individual_details is not None:
+            self.__add_individuals_to_phenotips_from_ped(individual_details,project_id)
+          else:
+            if indiv_id_list is not None:
+              self.__add_individuals_to_phenotips_from_vcf(indiv_id_list,project_id)
+            else:
+              print '\nno information in VCF to create patients with\n'
+              sys.exit()
+        else:
+          print '\n----not adding these patients to local phenotips instance.----\n'
+            
 
     #given a list of individuals add them to phenotips
-    def __add_individuals_to_phenotips(self,individuals):
+    def __add_individuals_to_phenotips_from_vcf(self,individuals,project_id):
       '''given a list of individuals add them to phenotips '''
-      for individual in  individuals:
-        print individual
+      for individual in  individuals:        
+        create_patient_record(individual,project_id)
+        
+    #given a list of individuals add them to phenotips
+    def __add_individuals_to_phenotips_from_ped(self,individual_details,project_id,):
+      '''given a list of individuals add them to phenotips '''
+      #for now only using gender information from the PED file.
+      for individual in  individual_details:
+        id=individual['indiv_id']
+        gender='F'
+        if individual['gender'] == 'male':
```

Comment:
to double check assumptions, would be good to do
if individual['indiv_id'] == 'female': gender='F' 
if individual['indiv_id'] == 'male': gender='M' 
else: raise ValueError